src/installer.ts

```diff
@@ -283,7 +283,14 @@ export class DotnetCoreInstaller {
       );
     }
 
-    return releasesInfo[0]['releases.json'];
+    const releaseInfo = releasesInfo[0];
+    if (releaseInfo['support-phase'] === 'eol') {
+      core.warning(
+        `${releaseInfo['product']} ${releaseInfo['channel-version']} is no longer supported and may not work properly.`

```

Comment:
```suggestion
        `${releaseInfo['product']} ${releaseInfo['channel-version']} is no longer supported and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the .NET support policy.`

```

Reply:
Incorporated your change and am rebuilding `index.js`.

Reply:
Resolved